src/features/checker/pages/ApplicationEvaluationOverviewPage/ApplicationEvaluationOverviewPage.tsx

```diff
@@ -43,29 +41,20 @@ export const ApplicationEvaluationOverviewPage = ({
     dispatch(goToReviewApplicationsAction());
   };
 
-  const syncAndRefresh = async () => {
```

Comment:
^ How would we trigger AI evaluation for a situation where the checker doesn't have an AI evaluation for the application ? Is the assumption from the UI that this will always be present ?

Reply:
For now it is happening each time `before` fetching the checker data